contracts/StablecoinConverter.sol

```diff
@@ -133,6 +142,7 @@ contract StablecoinConverter is EpochTokenLocker {
         uint128[] memory volumes,
         uint128[] memory prices,  //list of prices for touched token only
         uint16[] memory tokenIdsForPrice  // price[i] is the price for the token with tokenID tokenIdsForPrice[i]
+                                          // tokenId for fee token does not need to be send, as it is always 0
```

Comment:
I am not sure I understand this comment. May I suggest the following if my interpretation is correct?
```suggestion
                                          // fee token id not required since always 0
```